### ISSUE TITLE
Notification when new row added

### ISSUE THREAD

**Issue by edwardbattistini** [levinunnink/html-form-to-google-sheet#37]
#discussion

I've been really playing a lot today with your work ! Thank you so much it's a really elegant and simple way to receive forms data. 

Now I was wondering if anyone was able to trigger notifications when a row is added successfully? 

I've created triggers .onChange and .onEdit as google recommends but it only works when it's a client side event not AppsScript

I've also tried to use the mailapp.sendemail() function but it won't even do anything with it. 

If anyone has managed to make it work I'm running out of ideas.

**Comment by deepanshu88** [comment]
Yes I have managed to make email notification work. Check out the complete code here - https://www.listendata.com/2024/10/free-html-forms-with-googlesheets.html
